chef_master/source/chef_deprecations_client.rst

```diff
@@ -163,6 +163,10 @@ All Deprecations
     - Resource(s) in a cookbook collide with the same resource(s) now included in Chef.
     - XX.X
     - 15.0
+  * - `CHEF-27 </deprecations_locale_lc_all.html>`__
+    - Deprecation of lc_all from locale resource
+    - 15.0
+    - 15.1
```

Comment:
Let's plan to deprecate this in 16.0.

Reply:
for the deprecation version let's set to 14.12, and we will backport the deprecation into the Chef 14 release we will make in April/May. And set the second number to 16.0 as I commented/